app/views/admin/categories/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<%=  provide(:title,"All categories") %>
+	<div>
+		<h1 class=""><%= yield(:title) %></h1>
+
+		<%= link_to "NEW CATEGORIES", new_admin_category_path %>
```

Comment:
@YutakaYamasaki 
- Change text to "ADD NEW CATEGORY"
- Add a class here so that it looks better `btn btn-primary`